Update allowselfservicepurchase-powershell.md (#5834)
MicrosoftDocs/microsoft-365-docs: microsoft-365/commerce/subscriptions/allowselfservicepurchase-powershell.md

Update-MSCommerceProductPolicy -PolicyId AllowSelfServicePurchase -ProductId $product.ProductID -Enabled $false
```

> If there are multiple values for the product, you can run the command individually for each value like this.

Reviewer: ```suggestion
If there are multiple values for the product, you can run the command individually for each value as follows:
```
